lib/commands/mongo.js

```diff
@@ -132,6 +139,307 @@ mongo.createUser = function (databaseName, username, password, isReadOnly, cb) {
 }
 
 // ------------------------------------------------------------------------------
+mongo.migrate = function (databaseName, version, cb) {
+  mongoController.databasesForUser(
+    userConfig.data.userId,
+    function (err, databases) {
+      var dbExists = false
+
+      if (err) {
+        err = error.handleApiError(err, 'GET_DATABASES', cb)
+        if (err.length > 0) {
+          return cb(err)
+        }
+      }
+      if (databases.length === 0) {
+        xervo.io.error('You currently have no databases. One can be created using the `mongo create` command.')
```

Comment:
You can actually pass that message to the callback as the error and it will print. Saves 1 line of code 🥂